content/docs/get-started/azure/deploy-changes.md

```diff
@@ -24,8 +24,8 @@ Pulumi computes the minimally disruptive change to achieve the desired state des
 Previewing update (dev):
 
      Type                      Name                     Plan       Info
-     pulumi:pulumi:Stack       update-az-templates-dev
- ~   └─ azure:storage:Account  storage                  update     [diff: ~enableHttpsTrafficOnly]
+     pulumi:pulumi:Stack       quickstart-dev
+ ~   └─ azure:storage:Account  storage                  update     [diff: ~tags]
 
 Resources:
     ~ 1 to update
```

Comment:
There is a string on line 40 that I can't comment on directly, but it says "Pulumi will update the storage account to enable the HTTPS-only enforcement. The resource group hasn't changed so it remains as-is."

That should be changed since we are no longer doing the HTTPS-only change.

Reply:
Thanks for pointing that out!